skins/laika/src/components/shared/Postal.vue

```diff
@@ -92,6 +125,15 @@ export default Vue.extend( {
 		zipValueEqualsPlaceholder(): boolean {
 			return this.$props.formData.postcode.value === this.$t( 'donation_form_zip_placeholder' );
 		},
+
+		filteredCountries(): Array<Country> {
+			return this.countries.filter( ( countryOption: Country ) => {
+				return countryOption.countryFullName
+					.toString()
+					.toLowerCase()
+					.indexOf( this.$data.countryInput.toLowerCase() ) >= 0;
+			} );
+		},
```

Comment:
Why is this method necessary? I'd expect the autocomplete to whittle down the full list based on the current input.

Reply:
buefy's autocomplete docs demonstrate it like that. we need our own filter